### PR TITLE
chore: bump moby/buildkit to v0.15.1

### PR DESCRIPTION
**Pull request by DmitriyMV**
That's it.

### CODE REVIEW

File: internal/config/constants.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ const (
 
 	// BuildKitContainerVersion is the version of buildkit container image.
 	// renovate: datasource=docker versioning=docker depName=moby/buildkit
-	BuildKitContainerVersion = "v0.15.0"
+	BuildKitContainerVersion = "v0.15.1"
```
Comment (smira) [inline internal/config/constants.go:24]: `make rekres` is missing

Comment (DmitriyMV) [reply internal/config/constants.go:24]: Nope. No occurence of `moby/buildkit` in the kres module, so no update required. Running `make kres && ./_out/kres-linux-amd64 gen` also gives nothing.

Comment (smira) [reply internal/config/constants.go:24]: ah, ok... I guess we're using default from the CI? @frezbo should we bump there as well?

Comment (frezbo) [reply internal/config/constants.go:24]: yeh, let's update in https://github.com/siderolabs/build-container/ too

Comment (DmitriyMV) [reply internal/config/constants.go:24]: Done in https://github.com/siderolabs/build-container/pull/118

**Pull request merged**